crud.py

```diff
@@ -0,0 +1,49 @@
+from datetime import date
+from sqlalchemy.orm import Session
+
+from db import models
+from schemas import AuthorCreate, BookCreate
+
+
+def create_author(db: Session, author: AuthorCreate):

```

Comment:
Consider adding type hints for the return type of the function, e.g., `-> models.Author`.